arm-sql/2015-05-01-preview/swagger/failoverGroups.json

```diff
@@ -204,194 +253,155 @@
           }
         },
-        "deprecated": false
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        },
+        "x-ms-examples": {
+          "List failover group": {
+            "$ref": "../examples/FailoverGroupList.json"
+          }
+        }
       }
     },
     "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Sql/servers/{serverName}/failoverGroups/{failoverGroupName}/failover": {
       "post": {
         "tags": [
           "FailoverGroups"
         ],
-        "description": "Sets which server is primary by failing over from the current primary server.",
+        "description": "Fails over from the current primary server to this server.",
         "operationId": "FailoverGroups_Failover",
-        "x-ms-examples": {
-          "Planned failover failover group": {
-            "$ref": "../examples/FailoverGroupFailover.json"
-          }
-        },
         "parameters": [
           {
-            "$ref": "#/parameters/ApiVersionParameter"
```

Comment:
nit: maintain a consistent parameter ordering, either place ApiVersion and SubscriptionId together at the beginning of the parameters section or at the end.